script.js

```diff
@@ -0,0 +1,23 @@
+let d = document; 
```

Comment:
I suggest to rename this file to `adaptive-header.js` because it is part of adaptive-header page

Reply:
thanks, renaimed file